lib/elasticsearch/elastic-query-builder.js

```diff
@@ -488,6 +488,57 @@ class ElasticQueryBuilder {
     }
   }
 
+  buildPackedFieldClause (value, field) {
+    // Figure out the base property (e.g. 'owner')
+    const baseField = field.replace(/_packed$/, '')
+    // Allow supplied val to match against either id or value:
+    return {
+      bool: {
+        should: [
+          { term: { [`${baseField}.id`]: value } },
+          { term: { [`${baseField}.label`]: value } }
+        ]
+      }
+    }
+  }
+
+  buildMultiFieldClause (value, fields) {
+    return {
+      bool:
+        { should: fields.map(field => ({ term: { [field]: value } })) }
+    }
+  }
+
+  // This builds a filter cause from the value:
+  buildClause (value, field) {
```

Comment:
Seems too general a name for the specific thing it's doing? Could it be:

`buildFilterClause (value, fieldConfig)`